src/Microsoft.Azure.WebJobs.Extensions.EventHubs/Config/EventHubWebJobsBuilderExtensions.cs

```diff
@@ -18,7 +18,7 @@ public static IWebJobsBuilder AddEventHubs(this IWebJobsBuilder builder)
                 throw new ArgumentNullException(nameof(builder));
             }
 
-            builder.AddEventHubs(p => {});
+            builder.AddEventHubs(ConfigureOptions);
```

Comment:
The `InitialOffsetProvider` settings come from `hots.json`. This means the settings would be applied to all EventHub triggers in the app. Do we want to introduce attribute settings for individual triggers?

Reply:
This is how other extensions like Service Bus work currently, options at the input source level apply to all functions in the app so it should be ok to continue this way.  If there is feedback, we can consider moving this to be at the function level via attributes.